pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -160,6 +160,9 @@ func defaultPredicates() sets.String {
 		// Fit is determined by node disk pressure condition.
 		factory.RegisterFitPredicate(predicates.CheckNodeDiskPressurePred, predicates.CheckNodeDiskPressurePredicate),
 
+		// Fit is determined by node pid pressure condition.
+		factory.RegisterFitPredicate(predicates.CheckNodePIDPressurePred, predicates.CheckNodePIDPressurePredicate),
+
```

Comment:
Should handle equiv cache invalidation properly when node condition changes. See: https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/factory/factory.go#L822-L827

Reply:
got that.

Reply:
done